.github/workflows/main.yml

```diff
@@ -34,6 +34,23 @@ jobs:
           - internal_investigation
           - spec
     name: "Ruby ${{ matrix.ruby }}: ${{ matrix.task }}"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "${{ matrix.ruby }}"
+          bundler-cache: true
+      - run: NO_COVERAGE=true bundle exec rake ${{ matrix.task }}
```

Comment:
[memo] Since there is no need to measure coverage for each version, it is disabled here.